bedrock/firefox/views.py

```diff
@@ -528,5 +528,8 @@ class HelloStartView(LatestFxView):
     def get_context_data(self, **kwargs):
         ctx = super(HelloStartView, self).get_context_data(**kwargs)
         incoming = self.request.GET.get('incomingConversation') or 'none'
+        source = self.request.GET.get('utm_content') or 'none'
+        source = source.replace('hello-tour_OpenPanel_', '')
```

Comment:
Also would be good to add some test for this like we do for `incomingConversation`: https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/tests/test_base.py#L1118